wandb/lightgbm/__init__.py

```diff
@@ -0,0 +1,29 @@
+'''W&B callback for lightgbm
+
+Really simple callback to get logging for each tree
+
+Example usage:
+
+param_list = [("eta", 0.08), ("max_depth", 6), ("subsample", 0.8), ("colsample_bytree", 0.8), ("alpha", 8), ("num_class", 10)]
+config.update(dict(param_list))
+lgb = lgb.train(param_list, d_train, callbacks=[wandb_callback()])
+'''
+
+import lightgbm
+import wandb
+
+
+def wandb_callback():
+    def callback(env):
+        eval_results = {}
+        recorder = lightgbm.record_evaluation(eval_results)
+        recorder(env)
+
+        print(eval_results)
```

Comment:
is this print debugging that is intended to stay, if you want to have a print in the callback i would suggest a debug option on wandb_callback and formatting the print output

Reply:
Fixed in [728](https://github.com/wandb/client/pull/728).